static/js/settings_org.js

```diff
@@ -111,9 +111,7 @@ exports.populate_auth_methods = function (auth_methods) {
         // Don't prepend a tip to custom emoji settings page. We handle it separately.
         $(".organization-box").find(".settings-section:not(.can-edit)")
             .not("#emoji-settings")
-            .prepend("<div class='tip'>" +
-                     i18n.t("Only organization administrators can edit these settings.") +
-                     "</div>");
+            .prepend(i18n.t("<div class='tip'>Only organization administrators can edit these settings.</div>"));
```

Comment:
A small tweak: It's better to leave markup like HTML out of the strings to be translated, where possible.

I just pushed 0986fe700 to do that here; instead of string concatenation we can use jQuery's DOM construction methods, to stay clear of the new lint rules.

Reply:
Cool, yeah, looks cleaner. Thanks.